PowerEditor/installer/nativeLang/french.xml

```diff
@@ -21,16 +21,16 @@ Translation note:
 					<Item menuId="macro" name="&amp;Macro"/>
 					<Item menuId="run" name="E&amp;xécution"/>
 					<Item menuId="Plugins" name="Modules d’ex&amp;tension"/>
-					<Item menuId="Window" name="&amp;Documents"/>
+					<Item menuId="Window" name="&amp;Onglets ouverts"/>
 				</Entries>
 				<!-- Sub Menu Entries -->
 				<SubEntries>
-					<Item subMenuId="file-openFolder" name="Ouvrir le répertoire du fichier en cours"/>
-					<Item subMenuId="file-closeMore" name="Fermer plusieurs documents"/>
+					<Item subMenuId="file-openFolder" name="Utiliser le chemin du fichier en cours et :"/>
+					<Item subMenuId="file-closeMore" name="Fermer plusieurs fichiers"/>
```

Comment:
> "fichiers" is not accurate term because the opened tabs are not necessary the existent files, they could be the new opened empty tab. That's why original English text is "Close Multiple Documents" but not "Close Multiple Files"

Once again if I agree with the wording "Documents" it is wiser not to talk about "files"